ateam_ui/src/src/overlay.ts

```diff
@@ -123,12 +168,57 @@ export class Overlay {
                 break;
             // MESH
             case 6:
-                // I think I can use a PIXI filter to do this more efficiently
+                // This is for 2d arrays of meshes
+                // ASSUMPTION MESH IS IN Y then X or row then column (row-major)
+                // Could just use flat if this was directly an array of arrays
+                let heat_texture = this.array_to_pixi_texture(this.mesh, this.mesh_alpha)
+                // console.log(heat_texture.valid);
```

Comment:
We should probably remove all the unused/commented out lines if they aren't necessary

Reply:
I would say I will get rid of the log but I should put it as a doc somewhere what the format is or else someone is going to get vonfused

Reply:
The descriptive comments are fine, I was referring to lines of code that have been commented out